angular/core/components/thread_preview/thread_preview.haml

```diff
@@ -3,7 +3,8 @@
     .thread-preview__icon
       %user_avatar{ng-if: "!thread.activeProposal()", user: "thread.author()", size: "medium"}
       .thread-preview__pie-container{ng-if: "thread.activeProposal()"}
-        %pie_chart.thread-preview__pie-canvas{votes: "thread.activeProposal().voteCounts", ng-if: "thread.activeProposal()"}
+        %pie_chart.thread-preview__pie-canvas{votes: "thread.activeProposal().voteCounts", ng-if: "thread.activeProposal() && lastVoteByCurrentUser(thread)", title: "{{ 'dashboard_page.thread_preview.you_voted' | translate:translationData(thread) }}"}
+        %pie_chart.thread-preview__pie-canvas{votes: "thread.activeProposal().voteCounts", ng-if: "thread.activeProposal() && !lastVoteByCurrentUser(thread)", title: "{{ 'dashboard_page.thread_preview.undecided' | translate }}"}
```

Comment:
You don't need a `thread.activeProposal()` check here because we're doing it already in the `.thread-preview__pie-container` element. 💛